packages/compass-shell/package.json

```diff
@@ -24,7 +24,7 @@
     "test-electron": "xvfb-maybe cross-env NODE_ENV=test karma start",
     "test-cov": "echo",
     "check": "npm run lint && npm run depcheck",
-    "lint": "eslint \"./src/**/*.{js,jsx}\" \"./test/**/*.js\" \"./electron/**/*.js\" \"./config/**/*.{js,jsx}\"",
+    "lint": "eslint \"./src/**/*.{js,jsx}\" \"./test/**/*.js\" \"./config/**/*.{js,jsx}\"",
```

Comment:
Can we also remove the `start` and `start:watch` commands? From here and the other places. Looks like it uses the `./config/webpack.dev.config.js` `devServer` option which we might be able to remove as well now? This could also happen in a later pr if we want, no need to address now if we want to keep these changes isolated.

Reply:
Yeah, makes sense, will do